configure.ac

```diff
@@ -370,10 +370,9 @@ AC_ARG_WITH(openssl,
     [AS_HELP_STRING([--with-openssl[[=PATH]]],
     [Specify OpenSSL path])], [], [with_openssl=yes])
 
-if  test -d /usr/local/Cellar/openssl/ && \
-    test -d /usr/local/opt/openssl/ && \
+if  test -d /usr/local/opt/openssl/ && \
```

Comment:
Before, it seemed quite clear that `/usr/local/Cellar/openssl` means Homebrew, does `/usr/local/opt/openssl` really have the same meaning? I am thinking that with this change, there could be many systems which have `/usr/local/opt/openssl`, but don't use brew, and the logic here would fail.

Reply:
Ok. I can add a more generic detection of homebrew. The problem was the specificity of then end of the path `openssl` versus some version I think. Will adjust. 🚀

Reply:
adjusted here and in core, will see how ci feels about the change, probably a-ok!